src/main/java/uk/ac/ebi/embl/api/validation/check/feature/CollectionDateQualifierCheck.java

```diff
@@ -85,57 +85,67 @@ public ValidationResult check(Feature feature)
 		for (Qualifier collectionQualifier : collectionDateQualifiers) 
 		{
 			String value = collectionQualifier.getValue();
-			if (value == null || value.isEmpty())
-			{
+			if (isValueEmpty(value)) {
 				reportError(collectionQualifier.getOrigin(),
```

Comment:
- Note to myself: adding an ERROR ValidationMessage in ```reportError```.
- Suggestion is to ```continue``` the loop here so that we skip any following code.